spring-web/src/main/java/org/springframework/http/ContentDisposition.java

```diff
@@ -48,6 +48,18 @@
 	private static final String INVALID_HEADER_FIELD_PARAMETER_FORMAT =
 			"Invalid header field parameter format (as defined in RFC 5987)";
 
+	/**
+	 * The {@literal attachment} content-disposition type.
+	 */
+	public static final String ATTACHMENT = "attachment";
+	/**
+	 * The {@literal form-data} content-disposition type.
+	 */
+	public static final String FORM_DATA = "form-data";
+	/**
+	 * The {@literal inline} content-disposition type.
+	 */
+	public static final String INLINE = "inline";
```

Comment:
> I have also kept the new constants as well, as they are still useful - for example, checking the type: disposition.getType().equals(ContentDisposition.ATTACHMENT).

Why don't we then add `isInline`, `isAttachment`, and `isFormData` methods then?